### PR TITLE
Add subscription plugin

### PR DESCRIPTION
**Pull request by p0lunin**

### CODE REVIEW

File: wallet/subscription.tact [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    var period: Uint32;
+    var start_time: Uint32;
+    var timeout: Uint32;
+    var last_payment_time: Uint32;
+    var last_request_time: Uint32;
+    var failed_attempts: Uint8;
+    var subscription_id: Uint32;
+
+    internal default() {
+        send(CarryMessageBalance)
```
Comment (oleganza) [inline wallet/subscription.tact:37]:
```suggestion
        send(InboundCoins) 
```
?

File: wallet/subscription.tact [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    var start_time: Uint32;
+    var timeout: Uint32;
+    var last_payment_time: Uint32;
+    var last_request_time: Uint32;
+    var failed_attempts: Uint8;
+    var subscription_id: Uint32;
+
+    internal default() {
+        send(CarryMessageBalance) 
+        self.benefeciary.fallback(
```
Comment (oleganza) [inline wallet/subscription.tact:38]:
```suggestion
        self.beneficiary.fallback(
```

File: wallet/subscription.tact (hunk 1 above)
Comment (p0lunin) [reply wallet/subscription.tact:37]: Nope.
https://github.com/ton-blockchain/wallet-contract/blob/main/func/simple-subscription-plugin.fc#L129-L134
https://github.com/ton-blockchain/wallet-contract/blob/main/func/simple-subscription-plugin.fc#L75-L79

Comment (p0lunin) [reply wallet/subscription.tact]: Ohh, I understand what you mean. Fixed in [478d31e](https://github.com/tact-lang/tact-bikeshed/pull/7/commits/478d31e269966436cd0128855fca11b5f53eda49)

Comment (p0lunin) [reply wallet/subscription.tact]: Yeah, there should be `WalletBalance`, not `MessageBalance`. Fixed in the [4b68128](https://github.com/tact-lang/tact-bikeshed/pull/7/commits/4b681286e9e397b293d0ad1e70a1bb9b634950a5)

File: wallet/subscription.tact [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+interface Beneficiary {
+    internal destruct#64737472(msg.sender: Plugin);
+    internal fallback#756e6b77(msg.sender: Plugin);
+    internal subscription#73756273(msg.sender: Plugin);
+}
+
+contract SubscriptionPlugin: Plugin {
+    var wallet: Wallet;
+    var benefeciary: Beneficiary;
```
Comment (oleganza) [inline wallet/subscription.tact:26]:
```suggestion
    var beneficiary: Beneficiary;
```
+ all other occurences

**Pull request merged**